### PR TITLE
lang: update translation.

### PR DESCRIPTION
**Pull request by Croden1999**
here is finished this pr #2005 .

### CODE REVIEW

**Review by Zangetsu38 — APPROVED** [review]

**Comment by Croden1999** [comment]
sorry, Mr.@Zangetsu38 , i pushed again, plz check again, thx.

File: lang/da.xml [hunk 1]
```diff
@@ -1,6 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<lang>	
+<lang>
```
Comment (Zangetsu38) [inline lang/da.xml:3]: why it said is changed ?

**Review by Zangetsu38 — APPROVED** [review]

File: lang/da.xml (hunk 1 above)
Comment (Croden1999) [reply lang/da.xml:3]: there has newline character.

Comment (Zangetsu38) [reply lang/da.xml:3]: ah oki

**Pull request merged**